public-lighting-demo-app/web-demo-app/pom.xml

```diff
@@ -111,6 +110,10 @@
       <artifactId>orika-core</artifactId>
     </dependency>
 
+   <dependency>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)